phpBB/language/en/install.php

```diff
@@ -530,6 +532,27 @@
 	'UPDATE_DATABASE_SCHEMA'		=> 'Updating database schema',
 	'UPDATE_FILES'					=> 'Update files',
 	'UPDATE_FILES_NOTICE'			=> 'Please make sure you have updated your board files too, this file is only updating your database.',
+	'UPDATE_INCOMPLETE'				=> 'Your phpBB installation has not been correctly updated.',
```

Comment:
This will only appear to somebody who has already downloaded and installed the latest package. So re-explaining to them where and how to get the latest package is redundant.

If the user deleted their install folder, they will get an error that it couldn't be found. So all you have to explain here is that users should visit the database_update script, and if it is missing, they should download the package version from phpbb.com again, upload the install folder, and then visit the script.